examples/user-config/full-set/sandpit-apps.yaml

```diff
@@ -65,6 +65,10 @@ library:
   aem_orchestrator_version: 1.0.1-SNAPSHOT
   aem_password_reset_version: 1.0.1
   oak_run_version: 1.4.15
+  aem_stack_manager_lambda_version: 0.0.1
```

Comment:
Same as the default config.

Reply:
Changed to: 
- aem_stack_manager_version: 2.0.0